Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ however, it has to be formatted properly to pass verification tests.
 
 ## [Unreleased]
 
+### Changed
+- Exposed `displayIndex` property for `Pointer`, `Touchscreen`, `TouchControl`, `TouchState`, `Mouse`, `MouseState`; Enable multi-display input for all platforms that implement it;
```

Comment:
I know this concept of displayIndex exist in Unity and has been reused in this package. It is a bit misleading since display in industry typically means a physical display panel or monitor, while this actually refer to a "logical screen". I wonder whether we should clarify or this, e.g. "Exposed 'displayIndex' property .... which enables looking up the associated logical screen associated with the event available via https://docs.unity3d.com/ScriptReference/Display.html". This would hopefully avoid confusion that the event could be traced back to a physical display panel which is not currently the case. Such tracability could in the future be made available by also providing a reference to e.g. DisplayInfo-like information, but such information would only apply to certain devices such as e.g. TouchScreen, EyeTracker etc. It might be good to call out in the property documentation for displayIndex that the result, or associated coordinates may be incorrect if the display is projected on multiple physical monitors, especially if their resolution or aspect ratio is different.
Out of interest did you test any scenario with "duplicated desktop" where 2 physical displays map to a single "logical screen"?

Reply:
So all together, you'd like it to read:
```
Exposed `displayIndex` property for `Pointer`, `Touchscreen`, `TouchControl`, `TouchState`, `Mouse`, `MouseState` which enables looking up the associated logical screen associated with the event available via https://docs.unity3d.com/ScriptReference/Display.html
```
Do I have that right?


Regarding the "duplicated desktop" question: I'll have to defer to @3iD0loN here. Joel did a lot of testing outside of the tests in this PR. @3iD0loN would you mind sharing if that's something you did?

Reply:
Actually, that's an interesting question, and interesting premise, I did not test if this would work on a mirrored display. Since there is no desktop implementation, I did not try, and for AndroidPlayer, there is a hard limit to 2 displays, so the only display that could be mirrored is Display 0. The only tests that could be conducted are for Editor, and those are sub-regions in the editor